db/buckets.go

```diff
@@ -35,7 +35,42 @@ const (
 	Peer // maps peer ID to peer multiaddresses
 )
 
+var bucketStrings = []string{
+	"StateTrie",
```

Comment:
why is Peer excluded in bucketStrings?

Reply:
this list was created before `db.Peer` was merged into main

Reply:
Actually I think we shouldn't maintain it but rely on existing tool called `stringer` it can generate string names for a type: https://go.dev/blog/generate 

I can add it with installation command in a separate PR

Reply:
Added here - https://github.com/NethermindEth/juno/pull/1958

Reply:
Done